### PR TITLE
Adds Keyboard Handwired alcor dactyl

### PR DESCRIPTION
**Pull request by rocketstrong600**
## Description

Adds a new handwired keyboard  alcor_dactyl that is a 5x7 Dactyl Manuform keyboard using a rp2040-zero

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/alcor_dactyl/info.json [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_alcor":{
```
Comment (zvecr) [inline keyboards/handwired/alcor_dactyl/info.json:29]:
```suggestion
        "LAYOUT":{
```

File: keyboards/handwired/alcor_dactyl/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_alcor(
```
Comment (zvecr) [inline keyboards/handwired/alcor_dactyl/keymaps/default/keymap.c:4]:
```suggestion
    [0] = LAYOUT(
```

File: keyboards/handwired/alcor_dactyl/readme.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# alcor_dactyl
+
+![alcor_dactyl](https://i.imgur.com/F0E1JCQh.jpg)
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/handwired/alcor_dactyl/readme.md:5]: Should be filled in or removed

File: keyboards/handwired/alcor_dactyl/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+* Keyboard Maintainer: [Ethan](https://github.com/rocketstrong)
+* Hardware Supported: *The PCBs, controllers supported*
+* Hardware Availability: *Links to where you can find this hardware*
```
Comment (zvecr) [inline keyboards/handwired/alcor_dactyl/readme.md:9]: Same here too

File: keyboards/handwired/alcor_dactyl/rules.mk [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# This file intentionally left blank
```
Comment (zvecr) [inline keyboards/handwired/alcor_dactyl/rules.mk:1]:
```suggestion
```

**Review by zvecr — APPROVED** [review]

**Comment by rocketstrong600** [comment]
Thanks for looking at it. i just tidied it up with your suggestions

File: keyboards/handwired/alcor_dactyl/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+//-----------------------------------------------------------------------
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+        KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+                KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO,
+                                KC_NO, KC_NO, KC_NO, KC_NO,
+                KC_NO, KC_NO,
+            KC_NO, KC_NO,
+            KC_NO, KC_NO
+    )
```
Comment (drashna) [inline keyboards/handwired/alcor_dactyl/keymaps/default/keymap.c:74]: This really should be formatted so that it's side by side, not vertical like this.

Comment (rocketstrong600) [reply keyboards/handwired/alcor_dactyl/keymaps/default/keymap.c:74]: is there away to do that without reordering the matrix in info.json

Comment (rocketstrong600) [reply keyboards/handwired/alcor_dactyl/keymaps/default/keymap.c]: i think it looks better now.

**Comment by rocketstrong600** [comment]
looks so much neater side by side

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/alcor_dactyl/readme.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# alcor_dactyl
+
+![alcor_dactyl](https://i.imgur.com/F0E1JCQh.jpg)
+
+*This is a handwired dactyl manufor utalising an RP2040-zero*
+
+* Keyboard Maintainer: [Ethan](https://github.com/rocketstrong)
+* Hardware Supported: *Waveshare RP2040-Zero*
+* Hardware Availability: *https://www.waveshare.com/wiki/RP2040-Zero*
+* 3D Print File Source: *https://dactyl.mbugert.de/manuform*
```
Comment (fauxpark) [inline keyboards/handwired/alcor_dactyl/readme.md:10]:
```suggestion
This is a handwired dactyl manufor utalising an RP2040-zero

* Keyboard Maintainer: [Ethan](https://github.com/rocketstrong)
* Hardware Supported: Waveshare RP2040-Zero
* Hardware Availability: https://www.waveshare.com/wiki/RP2040-Zero
* 3D Print File Source: https://dactyl.mbugert.de/manuform
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**